### DISCUSSION TITLE
hosting under a subdirectory

### DISCUSSION THREAD

**Discussion by naspert** [rackslab/Slurm-web#387] [Q&A]
Hello

Is it possible to host slurm-web under a website's subdirectory e.g.
https://myhost.com/slurm-web

Could not find any info in the help...
BR

**Comment by rezib** [comment]
Hello @naspert, to be honest I have never tested this so far but I *think* it doesn't work as I can think of some absolute paths, especially in the UI, eg. https://github.com/rackslab/Slurm-web/blob/main/frontend/src/plugins/runtimeConfiguration.ts#L32. I just opened #388 to track this feature request.

**Reply by naspert** [reply]
I managed to give it a try (the main host at our lab runs a reverse proxy that forwards requests according to network path), and it confirms that slurm-web ignores the subdirectory and forwards all requests to /
